.sphinx/_static/custom.css

```diff
@@ -376,3 +376,17 @@ p code.literal {
 .highlight .o {
     color: #BB5400;
 }
+
+.rubric > .hclass2 {
```

Comment:
Are you using the rest of the CSS in this file for the PDF?
If not, we could have a `pdf.css` file instead and keep the styling separate?

Reply:
The CSS here applies to the HTML output, not the PDF. Sphinx is a bit quirky when it comes to PDF output - as the recommended method for PDF generation is to ignore the index page - but we have important information on the index page. So what happens is that `toctree` content is nested underneath any additional headings on the index page. For example:

```
Example
=======

Something

ToC
---

.. toctree::

   some-page
```

Would result in a PDF with the chapters:

1. ToC
1.1 some-page h1

The way around this is to use the `rubric` directive, which creates a heading but not an entry in the TOC. But rubric headings have a specific style, so I've made this CSS so the rubric can be styled to replicate a H2 heading that would commonly be found in the index page:

```
Example
=======

Something

.. rubric:: :h2:`ToC`

.. toctree::

   some-page
```

There's a bridge between the directive role and the CSS class in the `rst_prolog`

Reply:
Gnaah, I hoped this is a problem we don't need to worry about when using the Furo theme (I've seen it a lot with other themes, and with Furo I've never had issues).

It means that we're now hiding the Table of Contents on the start page:
![image](https://github.com/user-attachments/assets/d3a7da81-8284-4e16-87ff-2377d60bc073)

I think this is acceptable as long as it happens **only** on the start page - but my suspicion is that we need that on every page where we have subpages and headings.

There's probably not much to be done about it - but it could lead to confusion.

Reply:
It's only required on the index page, and it's just to avoid the content nesting issue described above.